content/manager_webui/logs-events-page.md

```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
-![Show]({{< img "ui/ui-logsNevents-show.png" >}})
 
+{{% gsNote title="Note" %}}
+If it is unclear if your timestamp is valid, look for this <i class="fa fa-calendar"></i> icon. When it shows - the input is valid and the data is automatically changed to match the filter.<br>
+![time range]({{< img "ui/eventsPage/timeRange.png" >}})<br>
+{{% /gsNote %}}
+
+### Message Text
+To filter logs & events by a specific message's text you can free type the text and matching logs & events will be automatically updated.<br/>
+![message text filter]({{< img "ui/eventsPage/messageTextFilter.png" >}})<br>
+
+### Clear Filters
+To clear all of the filters you can click the 'Clear Filters' button.
+
+# Sorting
+To sort logs & events click on the desired table's header to switch the ordering between ascending, descending and none.
+![sorting]({{< img "ui/eventsPage/sorting.png" >}})<br>
+
+# Pagination
+50 logs & events are presented per page. You can use the pagination bar to scroll through the pages.
+![pagination]({{< img "ui/eventsPage/pagination.png" >}})<br>
+
+# The Log/Event
+By default the table is not presenting all of the log/event information. In order to reveal all of the log/event information simply click on the row itself.<br>
+An event example:
+![event example]({{< img "ui/eventsPage/event.png" >}})<br>
+
+A log example:
+![log example]({{< img "ui/eventsPage/log.png" >}})<br>
+
+{{% gsNote title="Note" %}}
+Not all logs/events have the same fields as there are some fields which are specific to logs and others which are specific to events.<br>
+For example `Logger` and `Log Level` fields are specific to logs and `Event Type` field is specific to events.
+{{% /gsNote %}}
+
+# Adjust The Table
+By default the logs & events table is presenting all of the columns but you can choose which columns you would like the table to present.
+Click on the <i class="fa fa-cog"></i> icon, a dropdown menu containing all of the table headers names is presented.<br>
+![choose columns]({{< img "ui/eventsPage/columnsOrganizer.png" >}})<br>
+
+Clicking on a header toggles if it should be presented. Additionally you can drag and drop headers to determine the ordering of the columns.<br>
+Here is an example of a customised table:
+![customized table]({{< img "ui/eventsPage/customizedTable.png" >}})<br>
+
+# Persistency
+While you adjust the table's view by filtering, sorting and paging through it, those preferences are automatically saved at the page's url.<br>
+For every unique set of table filtering, sorting and paging preferences there is a unique url to match it and when loading the page those preferences will be automatically loaded from the page's url, by thus adjusting the table's view.
+This for example allow you to refresh the page, bookmark the page, or send the current page's url as a link to a colleague without having to set the view's preferences all over again when loading the page.
```

Comment:
allows

refresh the page, bookmark the page, -> refresh or bookmark the page or send..